pallets/proof-of-stake/rpc/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2021 Mangata team
+
+use codec::Codec;
+use jsonrpsee::{
+	core::{async_trait, Error as JsonRpseeError, RpcResult},
+	proc_macros::rpc,
+	types::error::{CallError, ErrorObject},
+};
+pub use proof_of_stake_runtime_api::ProofOfStakeApi as ProofOfStakeRuntimeApi;
+// use proof_of_stake_runtime_api::{RpcAmountsResult, RpcAssetMetadata, XYKRpcResult};
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_core::U256;
+use sp_rpc::number::NumberOrHex;
+use sp_runtime::{
+	generic::BlockId,
+	traits::{Block as BlockT, MaybeDisplay, MaybeFromStr},
+};
+use sp_std::convert::{TryFrom, TryInto};
+use std::sync::Arc;
+
+#[rpc(client, server)]
```

Comment:
If possible, please add documentation to API, describing what it is doing and what is the expected result.